lib/theme-api-client.js

```diff
@@ -65,33 +65,35 @@ async function checkCliVersion({ storeUrl, currentCliVersion = PACKAGE_INFO.vers
 /**
  * @param {object} options
  * @param {string} options.variationId
- * @param {string} options.channelId
+ * @param {array} options.channelIds
  * @param {string} options.apiHost
  * @param {string} options.storeHash
  * @param {string} options.accessToken
  * @returns {Promise<any>}
```

Comment:
You are now returning array of promises. 
Rather use this function with Promise.all() (which I would recommend) or [for await .. of](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for-await...of)

Reply:
I refactored that part. Please take a look.